AlphaWallet/Settings/Types/RPCServers.swift

```diff
@@ -33,6 +33,30 @@ enum RPCServer: Hashable {
         }
     }
 
+    func magicLinkNetwork(url: String) -> String {
+        if url.contains(Constants.mainnetMagicLinkPrefix) {
```

Comment:
You can probably use something like and switch instead of if-else:

```
let url = URL(string: url) //optional though
url.host == "poa.awallet.io"
```